Source/RpmWebView/Private/RpmWebViewWidget.cpp

```diff
@@ -9,12 +9,12 @@
 
 static const TCHAR* LinkObjectName = TEXT("rpmlinkobject");
 static const TCHAR* ClearCacheParam = TEXT("clearCache");
-static const TCHAR* QuickStartParam = TEXT("quickStart");
 static const TCHAR* FullBodyParam = TEXT("bodyType=fullbody");
 static const TCHAR* HalfBodyParam = TEXT("bodyType=halfbody");
 static const TCHAR* SelectBodyTypeParam = TEXT("selectBodyType");
 static const TCHAR* GenderMaleParam = TEXT("gender=male");
 static const TCHAR* GenderFemaleParam = TEXT("gender=female");
+static const TCHAR* SourceParam = TEXT("source=unreal-avatar-creator");
```

Comment:
shouldn't this be unreal-webview ?

Reply:
makes more sense indeed, when I think about it. 😄 I'll update it.